test/state/state.cpp

```diff
@@ -136,11 +136,24 @@ std::variant<int64_t, std::error_code> validate_transaction(const Account& sende
 }
 
 void finalize(State& state, evmc_revision rev, const address& coinbase,
-    std::optional<uint64_t> block_reward, std::span<Withdrawal> withdrawals)
+    std::optional<uint64_t> block_reward, std::span<Withdrawal> withdrawals,
```

Comment:
Nit: maybe `withdrawals` the last parameter, because it's the newest invention and they are applied last in the code.

Reply:
Done.